pvm.py

```diff
@@ -35,116 +43,230 @@ def _init_logger():
 	handler.setFormatter(formatter)
 	logger.addHandler(handler)
 
+# Log system initiated
 _init_logger()
 _logger = logging.getLogger("PVM")
 _logger.info("Logging system initiated in %s", LOG_PATH)
 
-media = ""
-
 '''
 UDP command example:
 /PVM HH MM SS <command> <value>
 '''
 
+# Parse UDP command
 def parse_commands(*args):
-	global media
-	global VIDEO_PATH
+	global media1
+	global media2
+	global VIDEO_PATH_ONE
+	global VIDEO_PATH_TWO
 	global IS_FILE_SET
-	
-	# Parse time and add 3 seconds
-	hh, mm, ss = args[1], args[2], args[3]
-	time_str = str(hh) + ":" + str(mm) + ":" + str(ss)
-	next_time = datetime.strptime(time_str, "%H:%M:%S") + timedelta(seconds=3)
+	global canPause
+	global canStart
+	global displayNum
 
 	# Get command
 	command = args[4]
 
-	# If scheduled time is behind current time, return.
-	if next_time.time() <= datetime.now().time():
-		_logger.info("Command: %s failed because scheduled time(%s) is behind current time.", command, time_str)
-		return
+	# Only start and pause command need to delay 3s
+	if command == "start" or command == "pause":
+    	# Parse time and add 3 seconds
+		hh, mm, ss = args[1], args[2], args[3]
+		time_str = str(hh) + ":" + str(mm) + ":" + str(ss)
+		next_time = datetime.strptime(time_str, "%H:%M:%S") + timedelta(seconds=3)
 
-	# Log command and execute time.
-	_logger.info("Command: %s, execute time: %s.", command, next_time.time())
-	
-	# Wait until current time is equal to next_time
-	while True:
-		now = datetime.now()
-		now = now.replace(microsecond=0)
-		if now.time() >= next_time.time():
-			break
-		sleep(0.005)
+    	# If scheduled time is behind current time, return.
+		if next_time.time() <= datetime.now().time():
+			_logger.info("Command: %s failed because scheduled time(%s) is behind current time.", command, time_str)
+			return
+
+		# Log command and execute time.
+		_logger.info("Command: %s, execute time: %s.", command, next_time.time())
+		
+		# Wait until current time is equal to next_time
+		while True:
+			now = datetime.now()
+			now = now.replace(microsecond=0)
+			if now.time() >= next_time.time():
+				break
+			sleep(0.005)
 	
     # Get value
 	if len(args) == 6:
 		value = args[5]
 
+	# Catch failed parse command
 	try:
 		# File command
 		if command=="file":
-			_logger.info("File set: %s", PREFIX_PATH + value)
-			IS_FILE_SET = True
-			media = OMXPlayer(PREFIX_PATH + value, dbus_name='org.mpris.MediaPlayer2.omxplayer', args=['--loop'])
-			media.pause()
-			VIDEO_PATH = value
+			# define displayNum here
+			if "|" not in value:
```

Comment:
should you use the alternative `parse_value` function below

```suggestion
			if not values[1]:
```

and so on...